src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/cloud-watch-fallback.mdx

```diff
@@ -0,0 +1,149 @@
+    * `INFRA_ENABLED`: true
+    * `LOGGING_ENABLED`: true (if sending logs)
+
+If you're using [`serverless-newrelic-lambda-layers`](https://github.com/newrelic/serverless-newrelic-lambda-layers), set:
+
+    ```yaml
+    custom:
+        newRelic:
+            enableExtension: true
+            enableFunctionLogs: false
+            enableIntegration: true
+            cloudWatchFilter: "*"
+    ```
+
+If you're using the [`newrelic-lambda-cli`](https://github.com/newrelic/newrelic-lambda-cli), set:
+
+    * `newrelic-lambda integrations install --nr-account-id &lt;YOUR_ACCOUNT_ID> --nr-api-key &lt;YOUR_API_KEY> --enable-logs`
+    * `newrelic-lambda layers install --function &lt;name or arn> --nr-account-id &lt;YOUR_NEW_RELIC_ACCOUNT_ID>`
+    * `newrelic-lambda subscriptions install --function &lt;name or arn> --filter-pattern ""`
+
+## Payload fallback with optional extension logs [#two]
+
+This option provides the lowest cost path that ensures instrumented payloads always make it to New Relic. This option ships function logs through the New Relic Lambda extension, so if the extension fails to start up or crashes, function logs will be missing from New Relic.
+
+If you're sending function logs, make sure the extension is configured to do so. These logs will only be sent by the extension and there will not be a CloudWatch fallback for logs in order to avoid duplicates.
+
+### Configuration [#configuration-two]
+
+After adding a New Relic Lambda layer, the extension is enabled and has log shipping disabled by default. If you want to see function logs in New Relic, you'll need to enable extension sending of function logs with an environment variable.
+
+It's important to note that function logs are just that, logs that are recorded by the function during its invocation. The extension will not send Lambda runtime logs like `START`, `END`, and `REPORT` lines.
+
+Set [extension environment variables](https://github.com/newrelic/newrelic-lambda-extension/blob/main/config/config.go) on your function:
+
+    * `NEW_RELIC_LAMBDA_EXTENSION_ENABLED`: true (default)
+    * `NEW_RELIC_EXTENSION_SEND_FUNCTION_LOGS`: true
+
+Set your `aws-log-ingestion` function environment variables:
+
+    * `INFRA_ENABLED`: true
+    * `LOGGING_ENABLED`: false (disable log forwarding to avoid duplicates or set the CloudWatch subscription filter pattern to match `NR_LAMBDA_MONITORING` lines only)
+
+If you're using [`serverless-newrelic-lambda-layers`](https://github.com/newrelic/serverless-newrelic-lambda-layers), set:
+
+    ```yaml
+    custom:
+        newRelic:
+            enableExtension: true
+            enableFunctionLogs: true
+            enableIntegration: true
+            cloudWatchFilter: "NR_LAMBDA_MONITORING" (only send payloads)
+    ```
+
+If you're using the [`newrelic-lambda-cli`](https://github.com/newrelic/newrelic-lambda-cli), set: 
+
+    * `newrelic-lambda integrations install --nr-account-id &lt;YOUR_ACCOUNT_ID> --nr-api-key &lt;YOUR_API_KEY>`
```

Comment:
```suggestion
    * `newrelic-lambda integrations install --nr-account-id <YOUR_ACCOUNT_ID> --nr-api-key <YOUR_API_KEY>`
```

Reply:
The integration should be a one-time thing only: once your AWS account is integrated with New Relic, you don't need to create a new integration. All new functions in the account would be able to share it.